src/protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2018"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
this has not been published yet, so we can still keep 0.5.1

Reply:
Sure, I just needed to use this to force Cargo to compile my local changes, e.g. for fluvio-protocol-derive